scipy/stats/_multivariate.py

```diff
@@ -3850,3 +3851,368 @@ def rvs(self, dim, size=1, random_state=None):
 
 
 unitary_group = unitary_group_gen()
+
+
+_mvt_doc_default_callparams = \
+"""
+loc : array_like, optional
+    Location of the distribution. (default ``0``)
+shape : array_like, optional
+    Positive definite matrix of the distribution. (default ``1``)
+df : integer, optional
+    Degrees of freedom of the distribution; must be a positive whole number. (default ``1``) 
+allow_singular : bool, optional
+    Whether to allow a singular matrix. (default ``False``)
+"""
+
+_mvt_doc_callparams_note = \
+"""Setting the parameter `loc` to ``None`` is equivalent to having `loc`
+be the zero-vector. The parameter `shape` can be a scalar, in which case
+the shape matrix is the identity times that value, a vector of
+diagonal entries for the shape matrix, or a two-dimensional array_like.
+"""
+
+_mvt_doc_frozen_callparams_note = \
+"""See class definition for a detailed description of parameters."""
+
+mvt_docdict_params = {
+    '_mvt_doc_default_callparams': _mvt_doc_default_callparams,
+    '_mvt_doc_callparams_note': _mvt_doc_callparams_note,
+    '_doc_random_state': _doc_random_state
+}
+
+mvt_docdict_noparams = {
+    '_mvt_doc_default_callparams': "",
+    '_mvt_doc_callparams_note': _mvt_doc_frozen_callparams_note,
+    '_doc_random_state': _doc_random_state
+}
+
+
+class multivariate_t_gen(multi_rv_generic):
+    r"""
+    A multivariate t-distributed random variable.
```

Comment:
```suggestion
    A multivariate t-distributed random variable.
    
```
That should fix the rendering on the top level stats documentation page:
![image](https://user-images.githubusercontent.com/6570539/92642386-f522c680-f294-11ea-9ba3-780015e238b3.png)